pantheon-bundle/src/test/java/com/redhat/pantheon/model/api/ResourceDecoratorTest.java

```diff
@@ -0,0 +1,171 @@
+package com.redhat.pantheon.model.api;
+
+import com.redhat.pantheon.model.api.v2.ResourceDecorator;
+import org.apache.sling.api.resource.ModifiableValueMap;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.api.resource.ValueMap;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+
+/**
+ * TODO This class has a set of empty tests for methods which are already tested elsewhere.
+ * @author Carlos Munoz
+ */
+@ExtendWith({MockitoExtension.class})
+class ResourceDecoratorTest {
+
+    @Mock
+    Resource resource;
+
+    @Test
+    void createChild() {
+    }
+
+    @Test
+    void getChild() {
+    }
+
+    @Test
+    void getOrCreateChild() {
+    }
```

Comment:
I left a comment at the top of the class. They refer to the methods of the `ResourceDecorator` class. They are already tested elsewhere. I can remove them for clarity. Thoughts?

Reply:
I suppose they weren't hurting anything, but I see you already removed them. Your comment (which I missed originally) had said that the empty tests were to denote functionality that had been covered elsewhere. That seems like a good messaging technique for future developers - I would have only requested that you add *where* that other testing occurs.

Reply:
:smile: I pre-empted your comment. It is weird to have empty test methods, not a common practice (that I know)

Reply:
Ok. I don't feel strongly either way. This is such core functionality that if it ever changes - by accident or otherwise - the effects will be immediately apparent, whether it's explicitly tested in this class or whether we rely on implicit coverage from other tests. So I'm confident that we can't make a 'wrong' decision about this particular class and these particular tests.